src/detection/card/content/items/description.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Box, Typography } from '@material-ui/core'
+import { Description as DescriptionIcon } from '@material-ui/icons'
+
+import { useStyles } from 'detection/card/content/items/items-style'
+
+export const Description = () => <Box classes={useStyles()}>
+  <DescriptionIcon />
+  <Typography variant={'h6'} classes={useStyles().contentText}>
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/125928514-3f10191f-ae56-43b5-a5f1-274574bc0ced.png)
![image](https://user-images.githubusercontent.com/23569938/125928543-a7f443c4-f6f4-4f8e-ae82-1915bf221165.png)
![image](https://user-images.githubusercontent.com/23569938/125928567-da86e512-861d-4873-8ed6-9e42bd4bda91.png)
This `margin-left` prop is not applied, since it doesn't have a `root` field within its style, which is required when using the `classes` prop.